lms/static/js/learner_dashboard/spec/program_details_view_spec.js

```diff
@@ -493,10 +511,34 @@ describe('Program Details Header View', () => {
                 destination_url: 'industry.com',
             },
         ],
-        programTabViewEnabled: false
+        programTabViewEnabled: false,
+        isUserB2CSubscriptionsEnabled: false,
     };
     const data = options.programData;
 
+    const testSubscriptionState = (state, heading, body, trial = false) => {
+        view = initView({
+            programData: $.extend({}, options.programData, {
+                subscription_eligible: true,
+            }),
+            isUserB2CSubscriptionsEnabled: true,
+            subscriptionData: [
+                $.extend({}, options.subscriptionData[0], {
+                    subscription_state: state,
+                    ...(trial && {
+                        trial_end: moment().add(3, 'days').utc().format('YYYY-MM-DDTHH:mm:ss[Z]'),
```

Comment:
It's difficult to parse what is being spread here, is there a more clear way of writing this?

Reply:
sure, will change this to be more readable 
the above code is a common way to conditionally add a property to an object